WordPress/Classes/ViewRelated/NUX/CreateAccountAndBlogViewController.m

```diff
@@ -33,6 +37,7 @@ @interface CreateAccountAndBlogViewController ()<UITextFieldDelegate,UIGestureRe
     WPWalkthroughTextField *_emailField;
     WPWalkthroughTextField *_usernameField;
     WPWalkthroughTextField *_passwordField;
+    UIButton *_onePasswordButton;
```

Comment:
Already did that in the [upcoming 2FA pull request!](https://github.com/wordpress-mobile/WordPress-iOS/tree/issues/3223-multifactor-auth-support)